app/components/admin/sdg/targets/index_component.html.erb

```diff
@@ -0,0 +1,38 @@
+<% provide(:title) do %>
+  <%= t("admin.header.title") %> - <%= title %>
+<% end %>
+
+<h2><%= title %></h2>
+
+<%= link_to t("admin.sdg.goals.index.title"), admin_sdg_goals_path %>
+<%= link_to t("admin.sdg.targets.index.title"), admin_sdg_targets_path %>
+
+<table>
+  <thead>
+    <tr>
+      <th></th>
+      <th id="target-title" scope="col"><%= attribute_name(:title) %></th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% targets.group_by(&:goal).map do |goal, targets| %>
+      <tr class="goal-header">
+        <th id="<%= header_id(goal) %>" colspan="2" scope="colgroup">
+          <%= goal.code %>. <%= goal.title %>
```

Comment:
`erb-lint` is giving a warning because the `.` is not being translated; not sure about the best approach here :thinking:.

Reply:
Maybe we can add a method to the Goal model. Probably this will be handy to call it from some different component views. What do you think?

```
def full_title
  "#{code}. #{title}"
end
```

Reply:
We'd still have a potential translation issue where some languages use the `.` and some do not... but for now I guess it's OK :smile:. Either leaving it in the view or moving it to the model is fine with me :ok_hand:.

Reply:
I kept this version finally!